updated readme with fixed typos and errors

SandeshGC opened the pull request:
Hey @mitalrs , I have fixed some typos and errors in the README.md file and made it better to understand for everyone.

This fixes issue #16

mitalrs commented:
@SandeshGC 
i cant find any changes in to the README.md file that can i considered the your fixed something.
can you mention the points you fixed, if yes then mentioned in the comment section please.

SandeshGC commented:
The changes are:
- 'tamplete' word has been removed with 'template' which is the correct one.
- the numbers in the ordered list of instructions were incorrect. I have checked and corrected them so that they are more understandable to readers.
- I have modified the markdown syntax to display the instructions more properly.

You can view the new README.md file on the branch in my fork: https://github.com/SandeshGC/Short-Profile-With-HTML-CSS/tree/enhancing-readme
The purpose of the change is just to improve the readability of README file
